rebasehelper/tests/test_application.py

```diff
@@ -67,7 +67,23 @@ class TestApplication:
 
     cmd_line_args: List[str] = ['--not-download-sources', '1.0.3']
 
-    def test_application_sources(self, workdir):
+    @pytest.fixture
+    def config(self):
```

Comment:
The name is confusing, as the method doesn't return instance of `Config`. It should be called `make_config()` or so.